### PR TITLE
Merging dev into master

### PR DESCRIPTION
**Pull request by KirillCS**

### CODE REVIEW

File: FleaMarket/ApplicationConfigurations.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace FleaMarket
+{
+    public class ApplicationConfigurations
```
Comment (artiomzubel) [inline FleaMarket/ApplicationConfigurations.cs:6]: Please add XML docs for this class

File: FleaMarket/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -1,8 +1,59 @@
...
+                                }
+
+                                @if (item.TradeEnabled)
+                                {
+                                    @:exchange is available
+                                }
+                            </div>
+                            <div class="text-black-50 mb-2 limited-text line-clamp-2 hidden-sm">@item.Description</div>
+                            <div class="d-flex flex-row flex-wrap justify-content-between align-items-center">
+                                <h5>Price: @(item.PriceType == PriceType.Definite ? item.Price?.ToString("C2", CultureInfo.CreateSpecificCulture("en-US")) : item.PriceType.ToString())</h5>
```
Comment (artiomzubel) [inline FleaMarket/Views/Home/Index.cshtml]: Can we use invariant culture?

File: FleaMarket/wwwroot/css/site.css [hunk 3]
```diff
@@ -69,3 +69,65 @@ body {
...
+}
+
+.limited-text {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 1;
+    -webkit-box-orient: vertical;
+}
+.line-clamp-1 {
```
Comment (artiomzubel) [inline FleaMarket/wwwroot/css/site.css:94]: Please add whitespaces before the class name and after "}"

File: Models/ViewModels/Account/LoginViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace FleaMarket.ViewModels
+{
+    public class LoginViewModel
+    {
+        [Required(ErrorMessage = "Name is the required field")]
```
Comment (artiomzubel) [inline Models/ViewModels/Account/LoginViewModel.cs]: Move error messages to constants

**Review by artiomzubel — CHANGES_REQUESTED** [review]

File: Models/ViewModels/Account/LoginViewModel.cs (hunk 4 above)
Comment (KirillCS) [reply Models/ViewModels/Account/LoginViewModel.cs]: It will be done later

**Pull request merged**